ui-modules/blueprint-composer/app/components/dsl-editor/dsl-editor.js

```diff
@@ -368,7 +368,7 @@ export function dslEditorDirective($rootScope, $filter, $log, brUtilsGeneral, bl
             }
         }
 
-        return new Dsl(KIND.METHOD, 'component').param(new Dsl(KIND.STRING, targetEntity.id));
+        return new Dsl(KIND.TARGET, TARGET.COMPONENT).param(new Dsl(KIND.STRING, targetEntity.id));
```

Comment:
DSL of kind METHOD referring to COMPONENT target created on select by default, however, it did not make sense, I changed this to be kind TARGET. @tbouron, could confirm that this is correct? I did not find a test to break with this change.

Reply:
It's been a while since I touched that code. It looks functionally the same so that should work @algairim, but I cannot give you a 100% guarantee